R/EpivizTreeData-class.R

```diff
@@ -542,8 +542,12 @@ EpivizTreeData$methods(
     \\item{end}{End of feature range to query}
     }
     "
+    if ("tsne"  %in% names(metadata(.self$.object))) {
+      #print("Hello123")
+    }
     if (!("tsne"  %in% names(metadata(.self$.object)))) {
-      tsne <- Rtsne(t(as.matrix(assays(.self$.object)$counts)))
+      #print("Hello")
+      tsne <- Rtsne(t(as.matrix(assays(.self$.object)$counts)), perplexity=2)
       metadata(.self$.object)$tsne <- tsne
```

Comment:
after this line, set the colnames on tsne slot

so 

`rownames(metadata(.self$.object)$tsne) <- colnames(.self$.object)`